drafts/use-cases/3.obtaining-events.md

```diff
@@ -0,0 +1,110 @@
+## Entry point
+
+### Story
+As an application user
+I want to display all my events
+So I can find out what awaits me in the nearest future.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api/events");
```

Comment:
I'm fine either way. If you decide to leave it, it somehow needs to be made clear though how that relative URL is transformed to an absolute URL. Maybe we could change it to

    var client = new HydraClient("http://example.com/");